pkg/cluster/describe.go

```diff
@@ -97,7 +97,8 @@ func PrintClusterDesctipion(connection *sdk.Connection, cluster *cmv1.Cluster) e
 		"Cluster Admin: %t\n"+
 		"Creator:       %s\n"+
 		"Created:       %v\n"+
-		"Expiration:    %v\n",
+		"Expiration:    %v\n"+
+		"Shard:         %v\n",
```

Comment:
The shard details and its actual existence should be hidden for customers as they are not authorized to view it.